app/src/main/java/com/example/desafio/adapter/MyPageAdapter.kt

```diff
@@ -28,9 +29,14 @@ class MyPageAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
             }
         }
     }
+/*
+    fun getFragmentFavorite(): Fragment {
+
+        return FragmentFavorite()
+    }*/
 
     override fun getCount(): Int {
-        return 6
+        return 5
     }
 
     override fun getPageTitle(position: Int): CharSequence? {
```

Comment:
aqui você pode criar uma lista estática dentro do escopo de MyPageAdapter com todos essas categorias dai quando for pegar o texto pra exibir aqui você só faz :

```suggestion
return categoriesList[position]
```

Reply:
Feito
Uma duvida para deixar estatico é só colocar o código dentro de companion objetc , assim como o trecho abaixo ?

companion object {
        //Lista com as categorias dos filmes
        val categoriesList: List<String> =
            Arrays.asList("Todos", "Acao", "Aventura", "Drama", "Ficcao", "Comedia")
    }